pkg/storaged/nfs-panel.jsx

```diff
@@ -106,10 +106,10 @@ export class NFSPanel extends React.Component {
                                   { title:  _("Size"), transforms: [cellWidth(40)] }
                               ]}
                               rows={mounts} />
-                    : <EmptyState className='storage-card-empty-state'>
-                        <Title headingLevel="h5" size="md">
```

Comment:
This is the empty state we display from pkg/lib/cockpit-component-table.jsx as well.
I feel if we change it here we need to change it there as well.

Reply:
Ah, yeah, we might need to.